packages/reactivity/src/ref.ts

```diff
@@ -117,11 +117,11 @@ export function customRef<T>(factory: CustomRefFactory<T>): Ref<T> {
   return r as any
 }
 
-export function toRefs<T extends object>(object: T): ToRefs<T> {
-  if (__DEV__ && !isProxy(object)) {
+export function toRefs<T extends object | any[]>(object: T): ToRefs<T> {
+  if (__DEV__ && !isProxy(object) && !isArray(object)) {
```

Comment:
Why is this necessary? It should apply for reactive arrays only.

Reply:
Oh whoops. The change is reflected in the array but can't be watched / made reactive.
I've updated the PR to fix this.